SocialFilter/sections/specific.txt

```diff
@@ -1205,7 +1205,8 @@ babyclub.de##.social-links-container
 censor.net##.footer_theme
 diy.obi.ru##.rating-section-share
 artjuku.com,cinema-sketch.com,chiepedia.club,geek.sc##.fab-btn
-artjuku.com,knowl.jp,b-gamers.net,record-now.com,pinapopo.com,daeudaeu.com,sendaiminami-tusin.com,cinema-sketch.com,mhr-movie.com,proof0309.com,flatdmkobo.com,anime-torrent.com,trovivo.com,makuring.com,appl-y.com,taisy0.com,chiepedia.club,geek.sc##.sns-btn
+sdesignlabo.com,artjuku.com,knowl.jp,b-gamers.net,record-now.com,pinapopo.com,daeudaeu.com,sendaiminami-tusin.com,cinema-sketch.com,mhr-movie.com,proof0309.com,flatdmkobo.com,anime-torrent.com,trovivo.com,makuring.com,appl-y.com,taisy0.com,chiepedia.club,geek.sc##.sns-btn
+sdesignlabo.com###twitter-widget-0
```

Comment:
If I'm not wrong, `||platform.twitter.com/widgets.js` blocks also embedded Tweets.
I guess it can be added. if this website do no use any embedded Tweets in articles.

Reply:
@Yuki2718 Alex asked me to add the domains to the beginning of both rules, it's not that important, but I will move the domains to the beginning.

Reply:
@Yuki2718 in most cases we should do that :)